packages/http-client-csharp/generator/SamplePlugin/src/Program.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace SamplePlugin
+{
+    internal static class Program
+    {
+        public static int Main(string[] args)
+        {
+            return 0;
```

Comment:
this should not be needed for a plugin implementation

Reply:
It is needed. Otherwise all the dependencies don't get included in the output directory.